src/oic/oic/provider.py

```diff
@@ -1501,6 +1468,39 @@ def verify_redirect_uris(registration_request):
 
         return verified_redirect_uris
 
+    def _verify_sector_identifier(self, request):
+        """
+        Verify `sector_identifier_uri` is reachable and that it contains `redirect uri`s.
+
+        :param request: Provider registration request
+        :return: si_redirects, sector_id
+        :raises: InvalidSectorIdentifier
+        """
+        si_url = request["sector_identifier_uri"]
+        try:
+            res = self.server.http_request(si_url)
+        except ConnectionError as err:
+            logger.error("{}", err)
+            res = None
+
+        if not res:
```

Comment:
Probably not, if you look at the previous code, it was doing the same thing. This catches the case when the response doesn't raise an exception but returns `None`.